slurmci-test.toml

```
@@ -3,45 +3,49 @@
+  { file = "examples/DGmethods_old/ex_003_acoustic_wave.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "examples/DGmethods_old/ex_004_nonnegative.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "examples/Microphysics/ex_1_saturation_adjustment.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "examples/Microphysics/ex_2_Kessler.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/Euler/isentropicvortex.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/Euler/isentropicvortex-imex.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/Euler/isentropicvortex-multirate.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/compressible_Navier_Stokes/mms_bc_atmos.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/compressible_Navier_Stokes/mms_bc_dgmodel.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/compressible_Navier_Stokes/rising_bubble-model.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/compressible_Navier_Stokes/rising_bubble-model-imex.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods/compressible_Navier_Stokes/density_current-model.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/RTB_IMEX.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/isentropic_vortex_standalone.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/isentropic_vortex_standalone_IMEX.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/isentropic_vortex_standalone_aux.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/isentropic_vortex_standalone_bc.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/isentropic_vortex_standalone_integral.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/Euler/isentropic_vortex_standalone_source.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/compressible_Navier_Stokes/mms_bc.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/conservation/sphere.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "test/DGmethods_old/sphere/advection_sphere_lsrk.jl", slurmargs = ["--ntasks=2"], args = [] },
+  { file = "test/DGmethods_old/sphere/advection_sphere_ssp33.jl", slurmargs = ["--ntasks=2"], args = [] },
+  { file = "test/DGmethods_old/sphere/advection_sphere_ssp34.jl", slurmargs = ["--ntasks=2"], args = [] },
+  { file = "test/LinearSolvers/poisson.jl", slurmargs = ["--ntasks=2"], args = [] },
+  { file = "test/LinearSolvers/columnwiselu.jl", slurmargs = ["--ntasks=1"], args = []},
+  { file = "test/LinearSolvers/bandedsystem.jl", slurmargs = ["--ntasks=3"], args = [] },
```

CPU is timing out:
```suggestion
  { file = "test/LinearSolvers/bandedsystem.jl", slurmargs = ["--ntasks=3 --time=02:00:00"], args = [] },
```
or use more tasks?

GPU test is failing.

---

Currently working on tracking down the GPU bug